starlingcaptureapi/file_util.py

```diff
@@ -56,3 +58,37 @@ def digest_sha256(self, file_path):
                 hasher.update(byte_block)
             return hasher.hexdigest()
         # TODO: handle error (image not found, etc.)
+
+    def timestamp(self, file_path, ts_file_path, timeout=5, min_cals=2):
```

Comment:
I think `timestamp` alone masks the fact that there is a remote action behind this.

```suggestion
    def register_timestamp(self, file_path, ts_file_path, timeout=5, min_cals=2):
```